e2e/libs/keywords/volume_keywords.py

```
@@ -337,3 +339,22 @@ def get_volume_checksum(self, volume_name):
+        replica = replica_list[0]
+        replica_name = replica['metadata']['name']
+        node = self.node.get_node_by_name(node_name, namespace=LONGHORN_NAMESPACE)
+        for diskName in node.disks:
+            disk = node.disks[diskName]
+
+            for scheduledReplica in disk['scheduledReplica']:
+                if scheduledReplica == replica_name:
+                    logging(f"Found replica {scheduledReplica} on node {node_name} scheduled to disk {diskName}")
+                    return disk['storageMaximum']
+
+        assert False, f"Failed to find storageMaximum for volume {volume_name} replica {replica_name} on node {node_name}"
```

Ruff (0.8.0)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)